README.md

```diff
@@ -1 +1,277 @@
+[![Docker Stars](https://img.shields.io/docker/stars/linuxserver/doplarr.svg?color=94398d&labelColor=555555&logoColor=ffffff&style=for-the-badge&label=stars&logo=docker)](https://hub.docker.com/r/linuxserver/doplarr)
+[![Jenkins Build](https://img.shields.io/jenkins/build?labelColor=555555&logoColor=ffffff&style=for-the-badge&jobUrl=https%3A%2F%2Fci.linuxserver.io%2Fjob%2FDocker-Pipeline-Builders%2Fjob%2Fdocker-doplarr%2Fjob%2Fmain%2F&logo=jenkins)](https://ci.linuxserver.io/job/Docker-Pipeline-Builders/job/docker-doplarr/job/main/)
+[![LSIO CI](https://img.shields.io/badge/dynamic/yaml?color=94398d&labelColor=555555&logoColor=ffffff&style=for-the-badge&label=CI&query=CI&url=https%3A%2F%2Fci-tests.linuxserver.io%2Flinuxserver%2Fdoplarr%2Flatest%2Fci-status.yml)](https://ci-tests.linuxserver.io/linuxserver/doplarr/latest/index.html)
+
+[Doplarr](https://github.com/kiranshila/Doplarr) is an *arr request bot for Discord."
+
+[![doplarr](https://raw.githubusercontent.com/linuxserver/docker-templates/master/linuxserver.io/img/doplarr.png)](https://github.com/kiranshila/Doplarr)
+
+## Supported Architectures
+
+Our images support multiple architectures such as `x86-64`, `arm64` and `armhf`. We utilise the docker manifest for multi-platform awareness. More information is available from docker [here](https://github.com/docker/distribution/blob/master/docs/spec/manifest-v2-2.md#manifest-list) and our announcement [here](https://blog.linuxserver.io/2019/02/21/the-lsio-pipeline-project/).
+
+Simply pulling `lscr.io/linuxserver/doplarr` should retrieve the correct image for your arch, but you can also pull specific arch images via tags.
+
+The architectures supported by this image are:
+
+| Architecture | Tag |
+| :----: | --- |
+| x86-64 | amd64-latest |
+| arm64 | arm64v8-latest |
+
+## Application Setup
+
+Start by following the [Discord](https://github.com/kiranshila/Doplarr#discord) setup instructions from Doplarr's readme.
+
+- If you are using Overseerr, fill in the Overseerr API and URL variables, and leave the Radarr/Sonarr variables blank.
+- If you are using Radarr/Sonarr (and not using Overseerr), fill in the Radarr/Sonarr API and URL variables, and leave the Overseerr variables blank.
+
+Extra java options can be passed with the JAVA_OPTS environment variable, eg `-e JAVA_OPTS="-Xmx256m -Xms256m"`.
+
+Review and adjust any [Optional Settings](https://github.com/kiranshila/Doplarr#optional-settings) from Doplarr's readme.
+
+## Usage
+
+Here are some example snippets to help you get started creating a container.
+
+### docker-compose (recommended, [click here for more info](https://docs.linuxserver.io/general/docker-compose))
+
+```yaml
+---
+version: "2.1"
+services:
+  doplarr:
+    image: lscr.io/linuxserver/doplarr
+    container_name: doplarr
+    environment:
+      - PUID=1000
+      - PGID=1000
+      - DISCORD__TOKEN=
+      - OVERSEERR__API=
+      - OVERSEERR__URL=http://localhost:5055
+      - RADARR__API=
```

Comment:
Do we want to keep this style of env vars?
Seems like some are with one underscore and some with 2

Reply:
The app uses double. The ones with single are specific to our image (JAVA_OPTS).